src/utils.ts

```diff
@@ -99,3 +96,11 @@ export function nodeVersionWarning() {
     }
   } catch (e) {}
 }
+
+/**
+ * Returns the result of attempting to `require` the project's `package.json`
+ * @returns the result of a `require()` statement
+ */
+export const getPackageJson = () => {
+  return require('./package.json');
```

Comment:
Note the usage of '.' here, as opposed to '..' in [the old version](https://github.com/ionic-team/create-stencil/pull/76/files#diff-c1a80fbb2e96a42cd35c5a1e7c3d22df4a225970075327f9a65ddab9510f06a8L2)